ui/addhardware.ui

```diff
@@ -1318,12 +1318,6 @@
                                   <object class="GtkComboBox" id="panic-model">
                                     <property name="visible">True</property>
                                     <property name="can-focus">False</property>
-                                    <property name="has-entry">True</property>
-                                    <child internal-child="entry">
-                                      <object class="GtkEntry">
-                                        <property name="can-focus">True</property>
-                                      </object>
-                                    </child>
```

Comment:
I would drop this change. We should still allow users to type in the model manually in case they connect to libvirt that doesn't report panic models.